### ISSUE TITLE
Date Increment/Decrement not working on windows

### ISSUE THREAD

**Issue by jmzagorski** [dhruvasagar/vim-dotoo#84]
I tested two systems with my same vimrc & plugins:

GVIM 8.2 patches 1-1252 Windows 10
VIM 8.2 patches 1-1238 Debian bullseye (Windows Subsystem for Linux)

`<C-A>` and `<C-X>` do not work on the windows sys, but works in WSL. You can see the two gifs below that show the issue. The dark background is debian, and the cursor jumps to the end and starts incrementing the date. The light background is windows and the cursor jumps to the end and does nothing.. Are you able to confirm this on windows as well? I have not had too much time to dig into this, but wanted to get the issue posted. Is there specific debug information that may help?

![2020-07-19_21-33-06](https://user-images.githubusercontent.com/8404957/87891314-84cf9580-ca07-11ea-96cd-b6abffa2cbfb.gif)

**Comment by dhruvasagar** [comment]
Can you tell me the output of `echo dotoo#time#new().to_string()` on windows ?

**Comment by jmzagorski** [comment]
1969-12-31 Wed. 19:00, which seems related to https://github.com/dhruvasagar/vim-dotoo/issues/40. When I put in the fix suggessted by @christianhaargaard in that issue in `autoload/dotoo/time.vim`, the echo changes to 2020-07-20 Mon 06:38, but the increment/decrement does not work. Maybe there are multiple places that a similar fix?

**Comment by dhruvasagar** [comment]
@jmzagorski Thanks, kindly test using the latest changes and let me know.

**Comment by jmzagorski** [comment]
`echo dotoo#time#new().to_string()` works now, thanks! The increment/decrement mapping still does not work on windows for me.

**Comment by dhruvasagar** [comment]
@jmzagorski 

* How did you record that gif ? I am guessing you combined 2 separate gifs ?
* Tell me, does vim native `<C-A>/<C-X>` work in other buffers on numbers ?
* Are you using Gvim or terminal vim ? If terminal vim, what terminal are you using in windows ?
* Have you also checked the same mappings in a regular dotoo file, instead of the capture window ?

**Comment by jmzagorski** [comment]
_How did you record that gif ? I am guessing you combined 2 separate gifs ?_
It is one gif. I recorded split screen with windows terminal & Debian to the left and gvim to the right

_Tell me, does vim native <C-A>/<C-X> work in other buffers on numbers ?_
Yes

_Are you using Gvim or terminal vim ? If terminal vim, what terminal are you using in windows_
I tested in both vim and gvim on windows. I tested the older command shell, powershell and windows terminal. All do not work as expected

_Have you also checked the same mappings in a regular dotoo file, instead of the capture window ?_
I checked my `refile` and the cursor jumps to the year, changes it and then jumps to the seconds and does nothing (see gif)

![2020-07-20_14-03-38](https://user-images.githubusercontent.com/8404957/87970518-dc1a4800-ca91-11ea-8b08-05b43bf61e50.gif)
